spec/index.bs

```diff
@@ -7345,6 +7351,17 @@ dictionary GPUFragmentState : GPUProgrammableStage {
 
             Otherwise:
             - |colorState|.{{GPUColorTargetState/writeMask}} must be 0.
+    - [$Validating GPUFragmentState's total color attachment depth$](|device|, |descriptor|.{{GPUFragmentState/targets}}) succeeds.
+</div>
+
+<div algorithm>
+    <dfn abstract-op>Validating GPUFragmentState's total color attachment depth</dfn>({{GPUDevice}} |device|, sequence<{{GPUColorTargetState}}?> |targets|)
+
+    1. Let |formats| be an empty sequence<{{GPUTextureFormat}}>
```

Comment:
nit: I think this should be a `[=list=]` rather than a sequence since it's not really interfacing with WebIDL.